bdd/spec/023_http_spec.sh

```diff
@@ -61,6 +61,52 @@ Describe "@std/http"
     End
   End
 
+  Describe "fetch directly"
+    before() {
+      sourceToAll "
+        from @std/app import start, print, exit
+        from @std/http import fetch, Request, Response
+
+        on start {
+          const res = fetch(new Request {
+            method: 'GET',
+            url: 'https://raw.githubusercontent.com/alantech/hellodep/aea1ce817a423d00107577a430a046993e4e6cad/index.ln',
+            headers: newHashMap('Content-Length', '0'),
+            body: '',
```

Comment:
`GET`s with bodies always weird me out even if the content length is set to nil

Reply:
That's why you should just use `get(url)` for GET calls :)